modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -446,14 +378,39 @@ example, `payload.people.person.address.*street` and the example
 `payload.people.person.address.street` (from <<single_value_array>>) return the
 same results.
 
+.DataWeave Script:
+[source,dataweave,linenums]
+----
+%dw 2.0
+var myArrayOfKeyValuePairs = [ "aString": "hello", "aNum": 2, "aString" : "world" ]
+var myArrayOfObjects = [ { "aString": "hello" }, { "aNum": 2 }, { "aString" : "world" } ]
+output application/json
+---
+{
+    myKeyValueExample : myArrayOfKeyValuePairs.*aString,
+    myObjectExample :  myArrayOfObjects.*aString
+}
+----
+
+.Output:
+[source,json,linenums]
+----
+{
+  "myKeyValueExample": [ "hello", "world" ],
+  "myObjectExample": [ "hello", "world" ]
+}
+----
+
 
 [[descendants]]
-== Descendants Selector (`..keyName`)
+== Descendants Selector (`..myKey`)
 
-This selector is applied to the context using the form `..keyName` and
+The `..` selector acts on arrays and objects.
+
+This selector applies to the context using the form `..myKey`, and it
 retrieves the values of all matching key-value pairs in the sub-tree under
-the current context. Regardless of the hierarchical structure these fields
-are organized in, they are all placed at the same level in the output.
+the selected context. Regardless of the hierarchical structure of the fields,
+the output is returned at the same level.
 
 In this example, all of the fields that match the key `name` are placed in a
 list called `names` regardless of their cardinality in the tree of the input
```

Comment:
Will users know what "cardinality" means?

Reply:
we have not received feedback about it. It's in dictionary, and the previous paragraph explains it in plain terms. so risk seems minimal.